client/scss/components/_tweet.scss

```diff
@@ -0,0 +1,5 @@
+//* @define tweet
+
+.tweet {
+  letter-spacing: normal;
```

Comment:
We override letter-spacing site wide in the default body typography:

```scss
body {
  @include font('HNL5');

  @include respond-to('medium') {
    @include font('HNL3');
  }
}
```

which loosens up the Helvetica Neue. But it's off Twitter brand, so doesn't look right when applied to the tweet UI (see updated PR description).